packages/opencensus-web-instrumentation-zone/test/test-interaction-tracker.ts

```diff
@@ -0,0 +1,408 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'zone.js/dist/zone.js';
+import { tracing, Span } from '@opencensus/web-core';
+import {
+  InteractionTracker,
+  RESET_TRACING_ZONE_DELAY,
+} from '../src/interaction-tracker';
+
+describe('InteractionTracker', () => {
+  const interactionTracker = InteractionTracker.instance;
+  let incrementTaskCountSpy: jasmine.Spy;
+  let decrementTaskCountSpy: jasmine.Spy;
+  let onEndSpanSpy: jasmine.Spy;
+
+  beforeEach(() => {
+    interactionTracker.resetCurrentTracingZone();
```

Comment:
Why is this needed?

Reply:
Not longer needed as now the `InteractionTracker` does this every time the interaction finishes.